homeassistant/components/vesync/sensor.py

```diff
@@ -5,7 +5,7 @@
 from dataclasses import dataclass
 import logging
 
-from pyvesync.vesyncfan import VeSyncAirBypass
+from pyvesync.vesyncfan import VeSyncAirBypass, VeSyncHumid200300S
```

Comment:
In order to split this PR, you should be able to add support for `VeSyncHumid200300S` sensors separately from the `Humidifier` platform.

Reply:
As I mention in the review comment, humidifiers aren't detected as devices right now.  But, I can split this change out and update sensors after the humidifier support exists